### PR TITLE
Adapter dev/1.0 redesign `Time`

### PR DESCRIPTION
**Pull request by GuoLei1990**

### CODE REVIEW

File: docs/time.zh-CN.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+
+## 属性
+
+| 名称                                                         | 释义                                                         |
+| ------------------------------------------------------------ | ------------------------------------------------------------ |
+| [timeScale](${api}core/Time#timeScale)                     | 时间的缩放                                                   |
+| [maximumDeltaTime](${api}core/Time#maximumDeltaTime)       | 最大间隔，帧率过低或卡顿时, `deltaTime` 不会超过 `maximumDeltaTime` * `timeScale` |
+| [frameCount](${api}core/Time#frameCount)                   | 引擎启动后累计的帧数                                         |
+| [deltaTime](${api}core/Time#deltaTime)                     | 上一帧距离当帧的增量时间，以秒为单位                         |
+| [unscaledDeltaTime](${api}core/Time#unscaledDeltaTime)     | 上一帧距离当帧的增量时间，以秒为单位，并且忽略了 [timeScale](${api}core/Time#timeScale) 影响 |
```
Comment (cptbtptpbcptdtptp) [inline docs/time.zh-CN.md:23]: _actualDeltaTime

File: docs/time.md [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+
+## 属性
+
+| 名称                                                         | 释义                                                         |
+| ------------------------------------------------------------ | ------------------------------------------------------------ |
+| [timeScale](${api}core/Time#timeScale)                     | The scale of time                                            |
+| [maximumDeltaTime](${api}core/Time#maximumDeltaTime)       | The maximum delta time, when the frame rate is too low or stuck, `deltaTime` will not exceed `maximumDeltaTime` * `timeScale` |
+| [frameCount](${api}core/Time#frameCount)                   | The total number of frames since the start of the engine     |
+| [deltaTime](${api}core/Time#deltaTime)                     | The delta time in seconds from the last frame to the current frame |
+| [unscaledDeltaTime](${api}core/Time#unscaledDeltaTime)     | The unscaled delta time in seconds from the last frame to the current frame，ignore [timeScale](${api}core/Time#timeScale) impact |
```
Comment (cptbtptpbcptdtptp) [inline docs/time.md:23]: _actualDeltaTime

File: docs/time.zh-CN.md (hunk 1 above)
Comment (GuoLei1990) [reply docs/time.zh-CN.md:23]: done

**Review by cptbtptpbcptdtptp — APPROVED** [review]
+1

**Pull request merged**